stock_picking_batch_extended/models/stock_move.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2016 Cyril Gaudin, Camptocamp SA
+# Copyright 2018 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    has_move_lines = fields.Boolean(compute="_compute_has_move_lines")
```

Comment:
This deserves a separate module as it is a generic feature and not specific to batch extended

Reply:
I have generated another module for that